### PR TITLE
travis: Do not set CNAME for caja-actions and pluma-plugins

### PR DESCRIPTION
**Pull request by rbuj**

### CODE REVIEW

File: travis/gen-index.sh [hunk 1]
```diff
@@ -170,4 +170,12 @@ echo "</ul>" >> ${index_page}
 echo "</body>" >> ${index_page}
 echo "</html>" >> ${index_page}
 mv ${index_page} ${directory}
-echo ${name}.mate-desktop.dev > ${directory}/CNAME
+
+case ${name} in
+caja-actions|pluma-plugins)
+#   echo mate-desktop.github.io/${name} > ${directory}/CNAME
```
Comment (raveit65) [inline travis/gen-index.sh:176]: No need to set an cname when mate-desktop.github.io is used.
CNAME file doesn't exists in this case.
Cname is only used for custom dns.

Comment (rbuj) [reply travis/gen-index.sh]: done

Comment (raveit65) [reply travis/gen-index.sh]: Let see how it works https://github.com/mate-desktop/caja-actions/pull/16

**Review by raveit65 — APPROVED** [review]
LGTM

**Pull request merged**